inventory/environ.py

```diff
@@ -303,6 +303,13 @@ def prep_for_yaml_out(inventory):
         del inventory_to_dump["splunk"]["shc"]
     return inventory_to_dump
 
+def validate_splunk_default_password(inventory):
+    password = inventory["all"]["vars"]["splunk"]["password"]
+    if password and len(password) >= 8:
```

Comment:
but what if you change this configuration from 8 symbols to 12?

Reply:
I mean `minPasswordLength ` under https://docs.splunk.com/Documentation/Splunk/7.2.0/Admin/Authenticationconf

Reply:
I think it'd be safe to flag this as a warning that's sent to stdout and not actually stopping the installation?